sweet/benchmarks/cockroachdb/main.go

```diff
@@ -0,0 +1,611 @@
+// Copyright 2024 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build !wasm
+
+package main
+
+import (
+	"bytes"
+	"context"
+	"flag"
+	"fmt"
+	"github.com/pkg/errors"
+	"golang.org/x/benchmarks/sweet/benchmarks/internal/driver"
+	"golang.org/x/benchmarks/sweet/benchmarks/internal/server"
+	"golang.org/x/benchmarks/sweet/common/diagnostics"
+	"os"
+	"os/exec"
+	"regexp"
+	"runtime"
+	"strconv"
+	"strings"
+	"sync"
+	"sync/atomic"
+	"syscall"
+	"time"
+)
+
+const (
+	// Arbitrarily chosen to match the cockroachdb default.
+	basePort = 26257
```

Comment:
Is this thing going to run in a container primarily? Otherwise, we may need to ensure this port is not taken.

Reply:
The etcd benchmark also hardcodes a port, so I think it should be okay?

Reply:
Yeah, I saw that. Let's assume it's ok unless we're told otherwise :)